packages/rum/src/domain/record/serialize.spec.ts

```diff
@@ -684,6 +694,30 @@ describe('serializeDocumentNode handles', function testAllowDomTree() {
     }
   })
 
+  it('serializes a document with adoptedStyleSheets', () => {
+    if (!isAdoptedStyleSheetsSupported()) {
+      pending('no adoptedStyleSheets support')
+    }
+    const isolatedDom = createIsolatedDom()
+    const styleSheet = new isolatedDom.window.CSSStyleSheet()
+    styleSheet.insertRule('div { width: 100%; }')
+    isolatedDom.doc.adoptedStyleSheets = [styleSheet]
+    expect(serializeDocument(isolatedDom.doc, DEFAULT_CONFIGURATION, DEFAULT_SERIALIZATION_CONTEXT)).toEqual({
+      type: NodeType.Document,
+      childNodes: [jasmine.objectContaining({ type: NodeType.Element, tagName: 'html' })],
+      adoptedStyleSheets: [
+        {
+          cssRules: ['div { width: 100%; }'],
+          disabled: undefined,
+          media: undefined,
+        },
+      ],
+      id: jasmine.any(Number) as unknown as number,
+    })
+
+    isolatedDom.clear()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it is usually better to cleanup the test in a `afterEach` callback, because if the test throws, we would still cleanup the test and not interfer with other. You could wrap this test in a `describe` and write a local `beforeEach` / `afterEach` next to your test.